Adafruit_BME280.h

```diff
@@ -29,7 +29,7 @@
 /*=========================================================================
     I2C ADDRESS/BITS
     -----------------------------------------------------------------------*/
-    #define BME280_ADDRESS                (0x77)
+    #define BME280_ADDRESS                (0x76) // eBay-sourced modules often have SDO pulled low, for whatever reason, changing the address to 0x76.
```

Comment:
Of course. That's why I tried to reverse that change (my apologies for fat fingering git: this is my first pull request).

The change I was intending to submit was the one regarding the sensor sleep time, as the default option causes the sensor to self-heat and affects the temperature reading.